packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -47,9 +41,10 @@ export interface FormGroupProps extends Omit<React.HTMLProps<HTMLDivElement>, 'l
   helperTextInvalidIcon?: React.ReactNode;
   /** ID of the included field. It has to be the same for proper working. */
   fieldId: string;
-  /** Object for identifying whether the form group has multiple inputs for a single label, and
-   * whether the multiple inputs are radio inputs or not. */
-  multipleInputs?: MultipleInputsObject;
+  /** Sets the role of the form group if multiple inputs are being passed in. If multiple radio inputs
+   * are passed in use "radiogroup", otherwise use "group" when passing in multiple of any other input.
+   */
+  role?: 'group' | 'radiogroup';
```

Comment:
One last thought about this, would it be better to just type this as `string` and just conditionally check for `group` or `radiogroup` explicitly as needed, so that if a consumer wants a different role to apply they can do so?

cc @nicolethoen

Reply:
I agree - I think it can be string in case someone wants to explicitly set role=none or role=presentation or who knows what else

Reply:
In terms of the scope of the Form Group fix, if we allow any role to be passed in that might require another update in Core (right now I believe they're only applying either `group` or `radiogroup`). If the intent of a form group is to act more like a `fieldset` (which by default gets announced as a "group"), only allowing either of those two options seems like the better solution. If we'd want to allow any role to be passed, however, it might also be worth adding a disclaimer in the prop description about using other roles (basically a "be careful" sort of thing).

Reply:
I think that we should allow any role to be passed through for now at least, since like @nicolethoen mentioned yesterday FormGroup is currently often used in a non-fieldset-like fashion. I also wonder if any consumer might currently be applying a different role to a FormGroup for some reason, in which case limiting it to only `group`/`radiogroup` could be breaking.

You bring up good points though, and this seems like a topic that deserves further discussion and maybe a change in the next breaking change release?